v20.1/backup.md

```diff
@@ -304,6 +317,47 @@ INCREMENTAL FROM
 Note that only the backup URIs you set as the `default` when you created the previous backup(s) are needed in the `INCREMENTAL FROM` clause of your incremental `BACKUP` statement (as shown in the example). This is because the `default` destination for a locality-aware backup contains a manifest file that contains all the metadata required to create additional incremental backups based on it.
 {{site.data.alerts.end}}
 
+### Create an encrypted backup
+
+<span class="version-tag">New in v20.1:</span> To create a [encrypted backup](#encrypted-backups), use the `encryption_passphrase` option:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> BACKUP TO \
+'gs://acme-co-backup/test-cluster' \
+WITH encryption_passphrase = 'password123';
+~~~
+~~~
+        job_id       |  status   | fraction_completed | rows | index_entries | bytes
+---------------------+-----------+--------------------+------+---------------+---------
+  543214409874014209 | succeeded |                  1 | 2597 |          1028 | 467701
+(1 row)
+~~~
+
+Encrypted file:
```

Comment:
BACKUPs contain many files... is this the manifest file? In any case, since they're all binary gibberish whether they're plaintext or encrypted, i'm not sure if we need to include the content?